modules/video_coding/packet_buffer.cc

```diff
@@ -447,7 +447,7 @@ std::vector<std::unique_ptr<PacketBuffer::Packet>> PacketBuffer::FindFrames(
         }
       }
 #endif
-      if (is_h264 || full_frame_found) {
+      if (is_h264 || is_h265 || full_frame_found) {
```

Comment:
Fine with that but why 108 explicitly introduce the full_frame_found flag for non hevc/avc streams?

Reply:
It's introduced in https://webrtc-review.googlesource.com/c/src/+/260324.